java/org/contikios/cooja/plugins/MoteInformation.java

```diff
@@ -96,18 +95,14 @@ public MoteInformation(Mote m, Simulation s, Cooja gui) {
     mainPane.add(smallPane);
 
     smallPane = new JPanel(new BorderLayout());
+    label = new JLabel("Mote identifier");
```

Comment:
Is it not "Mote Type Identifier" that is shown rather than "Mote identifier" ?

Reply:
Yep, fixed!